src/renderer/components/dock/logs/controls.tsx

```diff
@@ -77,11 +93,10 @@ const NonInjectedLogControls = observer(({ openSaveFileDialog, model }: Dependen
           onChange={togglePrevious}
           className="show-previous"
         />
-        <Icon
-          material="get_app"
-          onClick={downloadLogs}
-          tooltip="Download"
-          className="download-icon"
+
+        <DownloadLogsDropdown
+          downloadVisibleLogs={downloadLogs}
+          downloadAllLogs={downloadAllLogs}
```

Comment:
How about:

```jsx
<div>
  <DropdownToggle dropdownId="download-logs-dropdown">
    <Button>Download...</Button>
  </DropdownToggle>

  <DropdownContents dropdownId="download-logs-dropdown">
    {() => [
       <Button key="download-all" onClick={model.downloadAll}>Download all</Button>,
       <Button key="download visible" onClick={model.downloadVisible}>Download visible</Button>
    ]}
  </DropdownContents>
</div>
```

In this way: 
1. We would have reusable `Dropdown` component
2. Button that toggles visibility of dropdown can be anywhere
3. Business logic for closing dropdown when item is clicked can be written to Dropdown, and not to DownloadLogs.

Actually what I'm defining here is something even more reusable. It doesn't have anything to with Dropdown. Dropdown is just one usage of it.

So maybe even:

```jsx
const Dropdown = ({ id, contentForToggle, contents }) => (
  <div class={styles.dropdown}>
    <ActivatableToggle activationId={id}>
      <Button>Download...</Button>
    </ActivatableToggle>

    <ActivatableContent activationId="download-logs-dropdown">
      {() => [
         <Button key="download-all" onClick={model.downloadAll}>Download all</Button>,
         <Button key="download visible" onClick={model.downloadVisible}>Download visible</Button>
      ]}
    </ActivatableContent>
  </div>
);
```
```jsx
<Dropdown
  id="download-logs-dropdown"
  contentForToggle={<Button>Download...</Button>}
  contents={
    {() => [
      <Button key="download-all" onClick={model.downloadAll}>Download all</Button>,
      <Button key="download visible" onClick={model.downloadVisible}>Download visible</Button>
    ])
  } 
/>
```